apps/server/src/modules/files-storage/files-storage.module.ts

```diff
@@ -2,22 +2,26 @@ import { DynamicModule, Module, NotFoundException } from '@nestjs/common';
 import { S3Client } from '@aws-sdk/client-s3';
 import { MikroOrmModule, MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs';
 import { Configuration } from '@hpi-schul-cloud/commons';
+import { ConfigModule } from '@nestjs/config';
+import { Dictionary, IPrimaryKey } from '@mikro-orm/core';
+
```

Comment:
+1 for atomatism.

Reply:
It is between package and path imports, because the package imports should be on top. We do not need this line, but it is faster to add a new package on right position.